python/hail/tests/utils.py

```diff
@@ -4,7 +4,7 @@
 
 
 def startTestHailContext():
-    hail.init(master='local[2]', min_block_size=0, quiet=True)
+    hail.init(master='local[2]', min_block_size=0, quiet=False)
```

Comment:
intentional?

Reply:
ah oops. I like quiet for python tests but some of the useful information in `_same` is silenced. I can try to add that to the exception it throws.